README.md

```diff
@@ -5,6 +5,12 @@
 
 This project is using Ignite UI for Angular components to demonstrate different scenarios related to the usage of each component. The project is tightly related to [Ignite UI DocFX Site Builder](https://github.com/IgniteUI/igniteui-docfx). All samples are part of the official component topics from our [website](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid.html).
 
+The project is divided into two two separate applications:
+
+- A `default application`, which uses all the [igniteui-angular](https://travis-ci.org/IgniteUI/igniteui-angular) components to demonstrate sample applications
+
+- A `line of business` application, which demonstrates enhanced application scenarios, using igniteui-angular-charts, igniteui-angular-excel library, igniteui-angular-sparklines combined with `igniteui-angular` components.

```

Comment:
Line of Business

Reply:
IMO it is better to describe it like the igniteui-angular has been complemented with the charts and excel libraries, smth like: "A 'Line of Business' application complemented with UI and functionality from the Ignite UI for Angular Charts and Ignite UI for Angular Excel libraries"